roles/cnf_cert/defaults/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 tnf_stage: "tests"
 tnf_check_commit_sha: false
-test_network_function_repo: https://github.com/test-network-function/cnf-certification-test
+test_network_function_repo: "git@github.com:test-network-function/cnf-certification-test.git"
```

Comment:
don't we need SSH keys to pull with `git`? 
```
[f5@f5-jumphost ~]$ git clone git@github.com:test-network-function/cnf-certification-test.git
Cloning into 'cnf-certification-test'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```
It works great when I use a server with an account that has a SSH key associated in github

Reply:
That's a good point. I checked some other places where we clone with the git module, it's everywhere via HTTPS + no credentials. 

Let's stick to that, I'll fallback to HTTPS and add an increased idle timeout then
```
environment:
  GIT_HTTP_MAX_REQUESTS: "4"
  GIT_HTTP_IDLE_TIMEOUT: "3600" 
```

Reply:
> Let's stick to that, I'll fallback to HTTPS and add an increased idle timeout then

Stick to ssh or https?

IMO, we should try to keep it https since the repo is public, otherwise we are making it hard to consume

Reply:
I'll fallback to HTTPS and add an increased idle timeout then